modules/nw-egress-ips-assign.adoc

```diff
@@ -49,11 +49,38 @@ $ oc apply -f <egressips_name>.yaml <1>
 egressips.k8s.ovn.org/<egressips_name> created
 ----
 
-. Optional: Save the `<egressips_name>.yaml` file so that you can make changes later.
+. Optional: Store the `<egressips_name>.yaml` file so that you can make changes later.
+
 . Add labels to the namespace that requires egress IP addresses. To add a label to the namespace of an `EgressIP` object defined in step 1, run the following command:
 +
 [source,terminal]
 ----
 $ oc label ns <namespace> env=qa <1>
 ----
 <1> Replace `<namespace>` with the namespace that requires egress IP addresses.
+
+.Verification
+
+* To show all egress IPs that are in use in your cluster, enter the following command:
++
+[source,terminal]
+----
+$ oc get egressip -o yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+# ...
+spec:
+  egressIPs:
+  - 192.168.127.10
+  - 192.168.127.11
+# ...
+----
++
+[NOTE]
+====
```

Comment:
Maybe this note would sit better under the command?

Reply:
Let's try it out :hmM: